src/vscode-dts/vscode.d.ts

```diff
@@ -9178,6 +9178,11 @@ declare module 'vscode' {
 		 */
 		export const shell: string;
 
+		/**
+		 * An {@link Event} which fires when the default shell changes.
+		 */
+		export const onDidChangeShell: Event<string>;
+
```

Comment:
This needs to move into a `vscode.proposed.envShellEvent.d.ts` file instead to go through our proposal/finalization process.